### ISSUE TITLE
qx create xyz and lint should not produce errors

### ISSUE THREAD

**Issue by dgtlrift** [qooxdoo/qooxdoo-compiler#641]
When creating a reference application and running lint on it - ie
```
qx create xyz
cd xyz
qx lint
```
There are numerous errors regarding `error: Expected linebreaks to be 'LF' but found 'CRLF' (linebreak-style) at` due to what I suspect is simply a windows vs rest-of-the-world thing when it comes to line endings.

**Comment by hkollmann** [comment]
Working with Windows or Linux?

**Comment by hkollmann** [comment]
You could run qx lint --fix

**Comment by derrell** [comment]
I think the point here is that `qx create xyz` should generate files that `qx lint` will not find errors with. Shouldn't matter where it is run from (Windows orLinux).

**Comment by hkollmann** [comment]
I agree - but wich system did @dgtlrift used?

**Comment by dgtlrift** [comment]
Windows 10

**Comment by dgtlrift** [comment]
Issue also arrises on Mint themed Ubuntu

```
Error: error: Opening curly brace does not appear on the same line as controlling statement (brace-style) at source/class/test/Application.js:37:5:
  35 |      */
  36 |     main : function()
> 37 |     {
     |     ^
  38 |       // Call super class
  39 |       this.base(arguments);
  40 |


error: Opening curly brace does not appear on the same line as controlling statement (brace-style) at source/class/test/Application.js:43:7:
  41 |       // Enable logging in debug variant
  42 |       if (qx.core.Environment.get("qx.debug"))
> 43 |       {
     |       ^
  44 |         // support native logging capabilities, e.g. Firebug for Firefox
  45 |         qx.log.appender.Native;
  46 |         // support additional cross-browser console. Press F7 to toggle visibility


error: Newline required at end of file but not found (eol-last) at source/class/test/Application.js:72:4:
  70 |     }
  71 |   }
> 72 | });
     |    ^


error: Opening curly brace does not appear on the same line as controlling statement (brace-style) at source/class/test/test/DemoTest.js:39:5:
  37 |      */
  38 |     testSimple : function()
> 39 |     {
     |     ^
  40 |       this.assertEquals(4, 3+1, "This should never fail!");
  41 |       this.assertFalse(false, "Can false be true?!");
  42 |     },


error: Opening curly brace does not appear on the same line as controlling statement (brace-style) at source/class/test/test/DemoTest.js:48:5:
  46 |      */
  47 |     testAdvanced: function ()
> 48 |     {
     |     ^
  49 |       var a = 3;
  50 |       var b = a;
  51 |       this.assertIdentical(a, b, "A rose by any other name is still a rose");


error: Newline required at end of file but not found (eol-last) at source/class/test/theme/Appearance.js:18:4:
  16 |   {
  17 |   }
> 18 | });
     |    ^


error: Newline required at end of file but not found (eol-last) at source/class/test/theme/Color.js:18:4:
  16 |   {
  17 |   }
> 18 | });
     |    ^


error: Newline required at end of file but not found (eol-last) at source/class/test/theme/Decoration.js:18:4:
  16 |   {
  17 |   }
> 18 | });
     |    ^


error: Newline required at end of file but not found (eol-last) at source/class/test/theme/Font.js:18:4:
  16 |   {
  17 |   }
> 18 | });
     |    ^


error: Newline required at end of file but not found (eol-last) at source/class/test/theme/Theme.js:21:4:
  19 |     appearance : test.theme.Appearance
  20 |   }
> 21 | });
     |    ^

```
